include/osquery/logger.h

```diff
@@ -154,15 +154,15 @@ class LoggerPlugin : public Plugin {
   /**
    * @brief Set the process name.
    */
-  void setName(const std::string& name) {
```

Comment:
NB: This was *shadowing* the existing `setName`, though no virtual, it may have caused confusion.